nmdc_runtime/site/repository.py

```diff
@@ -461,8 +514,51 @@ def repo():
         apply_metadata_in.to_job(**preset_normal),
         export_study_biosamples_metadata.to_job(**preset_normal),
         ensure_alldocs.to_job(**preset_normal),
+        run_uberon_ontology_load.to_job(
+            name="uberon_ontology_load",
+            config=unfreeze(
+                merge(
+                    run_config_frozen__normal_env,
+                    {
+                        "ops": {
+                            "load_ontology": {"config": {"source_ontology": "uberon"}}
```

Comment:
I added this merge to allow me to use the same load_ontology method with different default parameters.